lib/_debugger.js

```diff
@@ -514,9 +514,14 @@ Client.prototype.mirrorObject = function(handle, depth, cb) {
     // TJ's method of object inspection would probably be good for this:
     // https://groups.google.com/forum/?pli=1#!topic/nodejs-dev/4gkWBOimiOg
 
-    var propertyRefs = handle.properties.map(function(p) {
-      return p.ref;
-    });
+    var isBuffer = handle.text === '#<Buffer>' && handle.className === 'Uint8Array';
+
+    var propertyRefs = handle.properties.map((p) => p.ref);
+    if (isBuffer) {
+      // this is Node Buffer, no need to retrieve all values, Buffer.inspect prints only INSPECT_MAX_BYTES of them
+      // we'll leave INSPECT_MAX_BYTES + 1 so if actual size is bigger it's displayed with ... ellipses
```

Comment:
Style: wrap lines at 80 columns, capitalize and punctuate comments.  `make lint` (and `make test`) should pass.